src/main/scala/cromwell/backend/provider/local/LocalBackendActor.scala

```diff
@@ -88,9 +89,9 @@ class LocalBackendActor(task: TaskDescriptor) extends BackendActor with StrictLo
       val command = initiateCommand()
       logger.debug(s"Creating bash script for executing command: $command.")
       writeBashScript(command, executionDir)
-      notifyToSubscribers(new TaskStatus(Status.Created))
+      notifySubscribers(CreatedTaskStatus(stdout.toString, stderr.toString))
```

Comment:
Oh the engine treats these opaquely, they're just served up on the logs and metadata endpoints.  A consumer of these endpoints would need to know how to actually interpret them.